_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -31966,15 +31966,9 @@
 /area/chapel/office)
 "bfW" = (
 /obj/structure/rack,
-/obj/item/storage/fancy/candle_box,
-/obj/item/storage/fancy/candle_box{
-	pixel_x = -3;
-	pixel_y = 3
-	},
-/obj/item/storage/fancy/candle_box{
-	pixel_x = 3;
-	pixel_y = -3
-	},
+/obj/item/storage/fancy/candle_box/full,
+/obj/item/storage/fancy/candle_box/full,
+/obj/item/storage/fancy/candle_box/full,
```

Comment:
You wiped out their offsets. Why?

Reply:
When I checked them on the test server, they still had their off-sets. Don't know why it did that.

Reply:
Fix this, please.

Reply:
Is fixed.